### ISSUE TITLE
[Broken] youtube:tab returns 404, channel scraping not working

### ISSUE THREAD

**Issue by bani6809** [yt-dlp/yt-dlp#132]
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running yt-dlp version **2021.03.01**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
yt-dlp --ignore-config -i -o "%(uploader)s (%(uploader_id)s)/%(upload_date)s - %(title)s - (%(duration)ss) [%(resolution)s] [%(id)s].%(ext)s" --prefer-ffmpeg --merge-output-format mkv --write-sub --all-subs --convert-subs srt --add-metadata --write-description --write-annotations --write-all-thumbnails --write-info-json --cookies youtube-cookies.txt --proxy "" -v https://www.youtube.com/channel/UCca2961Ton2js_f9IDXK9Wg
[debug] Command-line config: ['--ignore-config', '-i', '-o', '%(uploader)s (%(uploader_id)s)/%(upload_date)s - %(title)s - (%(duration)ss) [%(resolution)s] [%(id)s].%(ext)s', '--prefer-ffmpeg', '--merge-output-format', 'mkv', '--write-sub', '--all-subs', '--convert-subs', 'srt', '--add-metadata', '--write-description', '--write-annotations', '--write-all-thumbnails', '--write-info-json', '--cookies', 'youtube-cookies.txt', '--proxy', '', '-v', 'https://www.youtube.com/channel/UCca2961Ton2js_f9IDXK9Wg']
[debug] Loading archive file None
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] yt-dlp version 2021.03.01
[debug] Python version 3.9.2 (CPython 64bit) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg 4.3.1-2021-01-01-full_build-www.gyan.dev, ffprobe 4.3.1-2021-01-01-full_build-www.gyan.dev
[debug] Proxy map: {}
WARNING: A channel/user page was given. All the channel's videos will be downloaded. To download only the videos in the home page, add a "/featured" to the URL
[youtube:tab] UCca2961Ton2js_f9IDXK9Wg:
[download] Downloading playlist: Trilogy Media - Videos
[info] Writing playlist metadata as JSON to: Trilogy Media (UCca2961Ton2js_f9IDXK9Wg)\NA - Trilogy Media - Videos - (NAs) [NA] [UCca2961Ton2js_f9IDXK9Wg].info.json
[info] Writing playlist description to: Trilogy Media (UCca2961Ton2js_f9IDXK9Wg)\NA - Trilogy Media - Videos - (NAs) [NA] [UCca2961Ton2js_f9IDXK9Wg].description
[youtube:tab] Downloading page 1
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #1)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #2)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #3)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #4)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #5)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #6)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #7)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #8)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #9)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #10)
ERROR: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
  File "c:\users\admin\appdata\local\programs\python\python39\lib\site-packages\yt_dlp\extractor\common.py", line 654, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\site-packages\yt_dlp\YoutubeDL.py", line 2784, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 523, in open
    response = meth(req, response)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 632, in http_response
    response = self.parent.error(
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 561, in error
    return self._call_chain(*args)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 494, in _call_chain
    result = func(*args)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 641, in http_error_default




yt-dlp --ignore-config -i -o "%(uploader)s (%(uploader_id)s)/%(upload_date)s - %(title)s - (%(duration)ss) [%(resolution)s] [%(id)s].%(ext)s" --prefer-ffmpeg --merge-output-format mkv --write-sub --all-subs --convert-subs srt --add-metadata --write-description --write-annotations --write-all-thumbnails --write-info-json --cookies youtube-cookies.txt --proxy "" -v https://www.youtube.com/channel/UC06HVrkOL33D5lLnCPjr6NQ
[debug] Command-line config: ['--ignore-config', '-i', '-o', '%(uploader)s (%(uploader_id)s)/%(upload_date)s - %(title)s - (%(duration)ss) [%(resolution)s] [%(id)s].%(ext)s', '--prefer-ffmpeg', '--merge-output-format', 'mkv', '--write-sub', '--all-subs', '--convert-subs', 'srt', '--add-metadata', '--write-description', '--write-annotations', '--write-all-thumbnails', '--write-info-json', '--cookies', 'youtube-cookies.txt', '--proxy', '', '-v', 'https://www.youtube.com/channel/UC06HVrkOL33D5lLnCPjr6NQ']
[debug] Loading archive file None
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] yt-dlp version 2021.03.01
[debug] Python version 3.9.2 (CPython 64bit) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg 4.3.1-2021-01-01-full_build-www.gyan.dev, ffprobe 4.3.1-2021-01-01-full_build-www.gyan.dev
[debug] Proxy map: {}
WARNING: A channel/user page was given. All the channel's videos will be downloaded. To download only the videos in the home page, add a "/featured" to the URL
[youtube:tab] UC06HVrkOL33D5lLnCPjr6NQ:
[download] Downloading playlist: Breaking Taps - Videos
[info] Writing playlist metadata as JSON to: Breaking Taps (UC06HVrkOL33D5lLnCPjr6NQ)\NA - Breaking Taps - Videos - (NAs) [NA] [UC06HVrkOL33D5lLnCPjr6NQ].info.json
[info] Writing playlist description to: Breaking Taps (UC06HVrkOL33D5lLnCPjr6NQ)\NA - Breaking Taps - Videos - (NAs) [NA] [UC06HVrkOL33D5lLnCPjr6NQ].description
[youtube:tab] Downloading page 1
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #1)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #2)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #3)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #4)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #5)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #6)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #7)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #8)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #9)
WARNING: [youtube:tab] HTTP Error 404. Retrying ...
[youtube:tab] Downloading page 1 (retry #10)
ERROR: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
  File "c:\users\admin\appdata\local\programs\python\python39\lib\site-packages\yt_dlp\extractor\common.py", line 654, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\site-packages\yt_dlp\YoutubeDL.py", line 2784, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 523, in open
    response = meth(req, response)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 632, in http_response
    response = self.parent.error(
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 561, in error
    return self._call_chain(*args)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 494, in _call_chain
    result = func(*args)
  File "c:\users\admin\appdata\local\programs\python\python39\lib\urllib\request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)

```

## Description

Looks like youtube may have changed the channel page format. We are now getting 404 when scraping channels, eg https://www.youtube.com/channel/UCca2961Ton2js_f9IDXK9Wg
Individual video downloads still works, eg https://www.youtube.com/watch?v=bddt-6i8SaI works.

This seems to have just started today.

**Comment by pukkandan** [comment]
@bani6809 Just made a new release fixing this

**Comment by bani6809** [comment]
confirmed fixed in 2021.03.03
